integration-tests/ocr2_test.go

```diff
@@ -62,8 +63,8 @@ func TestOCRBasic(t *testing.T) {
 	require.NoError(t, err, "Could not create private key from mnemonic")
 	logger.Info().Str("from", testAccount.String()).Msg("Funding nodes")
 
-	gasPrice := types.NewDecCoinFromDec("ucosm", types.MustNewDecFromStr("1"))
-	amount := []types.Coin{types.NewCoin("ucosm", types.NewInt(int64(10000000)))}
+	gasPrice := types.NewDecCoin("ucosm", math.NewInt(int64(10000000)))
```

Comment:
@EasterTheBunny the old apis are available with `Legacy` prefixes if that keeps this more readable.

Reply:
Ah ok. I just did a quick fix to get it working, but I can use the legacy stuff

Reply:
It looks like my change fixed the interface issues, but made the test not pass. I'll update it and push a new commit.